sentence_transformers/SentenceTransformer.py

```diff
@@ -430,6 +435,7 @@ def encode(
         convert_to_tensor: bool = False,
         device: str = None,
         normalize_embeddings: bool = False,
+        input_data_format: str = ImageChannelDimension.LAST,
```

Comment:
this should also be mentioned and explained in the docstring

Reply:
Fixed.